scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala

```diff
@@ -2268,7 +2316,7 @@ private[meta] class Parser(val input: Input)(implicit val dialect: Dialect) { pa
 
   def typeParam(ownerIsType: Boolean, ctxBoundsAllowed: Boolean): Type.Param = autoPos {
     if (token.is[Unquote]) return unquote[Type.Param]
-    var mods: List[Mod] = annots(skipNewLines = true)
+    var mods: List[Mod] = if (token.is[Ellipsis]) List(ellipsis(1, unquote[Mod])) else annots(skipNewLines = true)
```

Comment:
I think that's a problem, because the first line clearly fits the `tparam"..$mods $tpname[..$tparams] >: $tpeopt <: $tpeopt <% ..$tpes : ..$tpes"` specification.